Rakefile

```diff
@@ -8,18 +8,23 @@ Webapp::Application.load_tasks
 
 task(:export_log_data => :environment) do
 
-  CSV.open("orgs.csv","wb") do |csv|
-    csv << ["id","name","lat","lng","type"]
-    Location.where("region_id = ?",1).each{ |l|
-      csv << [l.id,l.name,l.lat,l.lng,l.is_donor ? "donor" : "recipient"]
+  CSV.open('orgs.csv', 'wb') do |csv|
+    csv << %w(id name lat lng type)
+    Location.where('region_id = ?', 1).each{ |l|
+      csv << [l.id, l.name, l.lat, l.lng, l.is_donor ? 'donor' : 'recipient']
     }
   end
 
   n = 0
+<<<<<<< HEAD
   CSV.open("logs.csv","wb") do |csv|
-    csv << ["id","date","volunteer_ids","donor_id","recipient_ids","part_ids",
-            "part_food_types","part_weights","part_counts",
-            "transport","scale","why_zero","schedule_id"]
+    csv << w%(id date volunteer_ids donor_id recipient_ids part_ids
+=======
+  CSV.open('logs.csv', 'wb') do |csv|
+    csv << %w(id date volunteer_ids donor_id recipient_ids part_ids
+>>>>>>> 1ada1b5... fixup
```

Comment:
This commit got a little messed up with the fixup. It looks like it gets sorted out in some other commit in the PR but it would be nice if we could clean this up a bit so this commit is solid by itself. If it would help, we can work together a bit some time this week to get this straightened out